readthedocs/projects/tasks.py

```diff
@@ -1207,61 +1200,6 @@ def move_files(
         target = version.project.rtd_build_path(version.slug)
         Syncer.copy(from_path, target, host=hostname)
 
-    if search:
```

Comment:
Overall, I'm fine removing this (large) block of code although I have no idea if there are any private installs using it. We probably want to make that clear in the changelog. 

Eventually however, it's likely that the entire `move_files` function can be removed as everything will be done with storage. I like this solution because it doesn't treat HTML as a special case separate from PDFs or ePubs which has been the source of some problems. We will also then be able to remove the syncers entirely.